pkg/operator/configobservation/apienablement/observe_runtime_config.go

```diff
@@ -17,8 +17,13 @@ import (
 
 var defaultGroupVersionsByFeatureGate = map[configv1.FeatureGateName][]groupVersionByOpenshiftVersion{
 	"MutatingAdmissionPolicy": {
-		{KubeVersionRange: semver.MustParseRange("< 1.34.0"), GroupVersion: schema.GroupVersion{Group: "admissionregistration.k8s.io", Version: "v1alpha1"}},
-		{KubeVersionRange: semver.MustParseRange(">= 1.34.0"), GroupVersion: schema.GroupVersion{Group: "admissionregistration.k8s.io", Version: "v1beta1"}},
+		// MutatingAdmissionPolicy was introduced as TechPreviewNoUpgrade in OCP 4.20 (k8s v1.33) with v1alpha1 API.
+		// OCP 4.21 (k8s v1.34) added v1beta1 API support alongside v1alpha1.
```

Comment:
Please mention that the upstream E2E feature tests exercise both versions, so pre-GA we must serve both.